Cargo.toml

```diff
@@ -33,8 +33,8 @@ mio = "^0.6.15"
 mio-extras = "^2.0.5"
 
 [target.'cfg(target_os="macos")'.dependencies]
-fsevent = "0.4"
-fsevent-sys = "2"
+fsevent = { git = "https://github.com/mtak-/fsevent-rust", branch = "downstream-test" }
+fsevent-sys = { git = "https://github.com/mtak-/fsevent-rust", branch = "downstream-test" }
```

Comment:
Hm, the pull request hasn't been reviewed yet. Should we ping the fsevent's author, or fork into notify-rs and use it?